cmd/podman/volume_prune.go

```diff
@@ -44,22 +45,29 @@ func init() {
 }
 
 func volumePrune(runtime *adapter.LocalRuntime, ctx context.Context) error {
-	prunedNames, prunedErrors := runtime.PruneVolumes(ctx)
-	for _, name := range prunedNames {
-		fmt.Println(name)
+	var (
+		errs    []*entities.VolumePruneReport
+		lastErr error
+	)
+	responses, err := runtime.PruneVolumes(ctx)
+	if err != nil {
+		return err
+	}
+	for _, r := range responses {
+		if r.Err == nil {
+			fmt.Println(r.Id)
+		} else {
+			errs = append(errs, r)
+		}
 	}
-	if len(prunedErrors) == 0 {
-		return nil
+	if len(errs) == 0 {
+		return lastErr
```

Comment:
This seems wrong. We never set lastErr?

Reply:
it was set to nil when declared.

Reply:
But nothing after this sets it to anything *other* than nil?

Reply:
Ah, I see it's set below... Hm.